Automattic/jetpack
# OpenTable: add fill in color for the block icon

#### Changes proposed in this Pull Request:

That color is important when displaying the icon inside the placeholder. When I changed the icon's color to monochrome, I failed to take that into account.

#### Testing instructions:

* Start with a site using the Beta version of blocks.
* Add an OpenTable block to a post, and check the look of the icon both in the block picker and inside the block placeholder:

![image](https://user-images.githubusercontent.com/426388/72431351-ee3ec500-3794-11ea-9101-5051aec93c34.png)

#### Proposed changelog entry for your changes:

* N/A
